source/blog/tag.html.erb

```diff
@@ -1,3 +1,4 @@
+<% add_atom_feed_to_head %>
```

Comment:
I'm happy to leave it as it is, was just curious to see what the normal/expected functionality would be